dadi/lib/controller/searchIndex.js

```diff
@@ -0,0 +1,44 @@
+const acl = require('./../model/acl')
+const config = require('./../../../config')
+const help = require('./../help')
+
+const SearchIndex = function (server) {
+  this.server = server
+
+  server.app.routeMethods('/api/index', {
+    post: this.post.bind(this)
+  })
+}
+
+SearchIndex.prototype.post = function (req, res, next) {
+  if (!req.dadiApiClient.clientId) {
+    return help.sendBackJSON(null, res, next)(
+      acl.createError(req.dadiApiClient)
+    )
+  }
+
+  // 404 if Search is not enabled
+  if (config.get('search.enabled') === false) {
```

Comment:
Should we be more strict and do `if (config.get('search.enabled') !== true) {`?

Reply:
done